src/text-input/src/TextInput.js

```diff
@@ -57,6 +69,13 @@ class TextInput extends PureComponent {
      */
     className: PropTypes.string
   }
+  constructor(props) {
```

Comment:
```
state = {
  value: ''
}
```

Should this take in account `defaultValue`?

Reply:
Got this part working